test/cookies/manifest.json

```diff
@@ -0,0 +1,131 @@
+{
+    "http_server": {
+        "addr": ":9999",
+        "dir": "../_res",
+        "testmode": false
+    },
+    "name": "check cookies from server as defined in test request",
+    "tests": [
+        {
+            "name": "set a sess cookie",
+            "request": {
+                "server_url": "http://localhost:9999",
+                "endpoint": "bounce-json",
+                "method": "POST",
+                "x-test-set-cookie": [
+                    {
+                        "name": "sess",
+                        "value": "my_sess"
+                    },
+                    {
+                        "name": "dummy",
+                        "value": "whatever",
+                        "path": "/mypath"
+                    }
+                ],
+                "body": {}
+            },
+            "response": {
+                "statuscode": 200,
+                "header": {
+                    "Set-Cookie": [
+                        "sess=my_sess",
+                        "dummy=whatever; Path=/mypath"
+                    ]
+                },
+                "cookie": {
+                    "sess": {
+                        "Name": "sess",
+                        "Value": "mysess"
+                    },
+                    "dummy": {
+                        "Name": "dummy",
+                        "Value": "whatever",
+                        "Path": "/mypath"
+                    }
+                },
+                "body": {}
+            },
+			"store_response_qjson": {
+				"sess_cookie": "cookie.sess"
```

Comment:
Is this documented? We need to document how to access the repsonse. is that simply the same object as the "response" definition in the test?